inst/examples/basic_kmplot2_ex.R

```diff
@@ -0,0 +1,17 @@
+data(adtte_sat)
+data(pseudo_ipd_sat)
+
```

Comment:
I think you need `library(survival)` here

Reply:
isn't survival already imported when importing maicplus?

Reply:
The package imports `survival` meaning that we can use `survival` functions in our package functions. It does not make make those functions available to the user the same as `library(survival)` does. (ie load and attaching the namespace).

To do that we would need to put `survival` in `depends:`, but this is not the modern way.